doc-content/enterprise-only/optimizer/optimizer-NP-complete-vs-NP-hard-con.adoc

```diff
@@ -7,9 +7,9 @@ The provided use cases are _probably_ http://en.wikipedia.org/wiki/NP-complete[N
 * There is no simple way to find the optimal solution of a problem in reasonable time.
 
 
-The implication is that solving your problem is probably harder than you anticipated, because the two common techniques will not suffice:
+The implication is that solving your problem is probably harder than you anticipated, because the two common techniques are not suffice:
```

Comment:
`are` doesn't seem right in this context, does it?

```suggestion
The implication is that solving your problem is probably harder than you anticipated, because the two common techniques do not suffice:
```